File: app/src/main/java/com/example/unlone/ui/Access/FirstAccessActivity.java
package com.example.unlone.ui.Access;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.unlone.R;

public class FirstAccessActivity extends AppCompatActivity {


    Button login, register;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_first_access);

        if (getSupportActionBar() != null){
            getSupportActionBar().hide();
        }

        login = findViewById(R.id.login);
        register = findViewById(R.id.register);

        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(FirstAccessActivity.this, LoginActivity.class));
            }
        });

        register.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(FirstAccessActivity.this, RegistrationActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/example/unlone/ui/access/FirstAccessActivity.kt
package com.example.unlone.ui.access

import android.os.Bundle
import android.view.Window
import androidx.appcompat.app.AppCompatActivity
import com.example.unlone.R

class FirstAccessActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_ACTION_BAR_OVERLAY)
        setContentView(R.layout.activity_first_access)
    }
}